### PR TITLE
Add release notes for v0.2.0

### PR DESCRIPTION
**Pull request by ivanayov**
Signed-off-by: Ivana Yovcheva <iyovcheva@vmware.com>

## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: Release.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+2. Deployment fully migrated to helm
+3. Dataplane is renamed to Forwarder. You now need to use `vpp-forwarder` instead of `vpp-dataplane`. For details, please see [Forwarder docs](https://github.com/networkservicemesh/networkservicemesh/blob/master/forwarder/README.md).
+4. API Changes: Unified {local,remote}.NetworkService API to simply NetworkService API
+
+## Bug fixes:
+1. The vpp-agent was not sending metrics previously. Fixed by [vpp-agent/PR#1495](https://github.com/ligato/vpp-agent/pull/1495)
+
+## Future changes:
+1. We are going to upgrade to Helm v3.0.0.
+2. Kind is going to be the only officially supported provisioning tool.
```
Comment (edwarnicke) [inline Release.md]: I don't recall deciding that kind would be the only officially supported provisioning tool.  Might be worth either discussing or refreshing my memory about the past discussion I've forgotten :)

Comment (nickolaev) [reply Release.md]: I guess it is a matter fo rewording, but now `kind` is where we default for trying NSM and is what we recommend. Vagrant is mainly for developer puprposes only

File: Release.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+5. **Expose Discovery Service to NSCs** \
+There will be occasions when NSCs (particularly those which are also NSEs) may need to make more sophisticated and programmatic decisions than can be expressed in the NS resource about Endpoint Selection. This allows them to discover the NSEs to which to apply that criteria. [PR#1825](https://github.com/networkservicemesh/networkservicemesh/pull/1825)
+
+
+## Backwards Compatibility:
+1. Updated Kubernetes to v1.16.2
+2. Deployment fully migrated to helm
+3. Dataplane is renamed to Forwarder. You now need to use `vpp-forwarder` instead of `vpp-dataplane`. For details, please see [Forwarder docs](https://github.com/networkservicemesh/networkservicemesh/blob/master/forwarder/README.md).
+4. API Changes: Unified {local,remote}.NetworkService API to simply NetworkService API
+5. We recommend using Kind as a provisioning tool. Vagrant is still suported, but is recommended mainly for developer purposes.
```
Comment (ivanayov) [inline Release.md]: @edwarnicke @nickolaev I reworded and moved it here

**Comment by nickolaev** [comment]
@edwarnicke I approved and letting you merge if you agree.

**Pull request merged**